src/pages/MyProfilePage.js

```diff
@@ -2,31 +2,67 @@ import React from "react";
 import "../styles.css";
 import { useSelector } from "react-redux";
 
-const getReseved = (rocket) => {
+const getReserved = (rocket) => {
   if (rocket.reserved) {
     return (
       <tr key={rocket}>
         <td className="separator">{rocket.rocket_name}</td>
       </tr>
     );
   }
+
+  return null;
+  // return (
+
+  // );
+};
+
+const getJoinedMission = (mission) => {
+  if (mission.active) {
+    return (
+      <tr key={mission}>
+        <td className="separator">{mission.mission_name}</td>
+      </tr>
+    );
+  }
+
   return null;
 };
+
 function MyProfilePage() {
   const { rockets } = useSelector((state) => state.rockets);
```

Comment:
- [ ] Please, I think it would be better here to use a filter instead of just using the condition in separate functions since it is better for Readability and Simplicity